README.md

```diff
@@ -105,3 +100,44 @@ http://<MINIKUBE_IP>/network-map-server/swagger-ui.html
 http://<MINIKUBE_IP>/single-line-diagram-server/swagger-ui.html
 http://<MINIKUBE_IP>/study-server/swagger-ui.html
 ```
+
+### Docker compose  deployment
+Clone deployment repository:
+```bash 
+git clone https://github.com/powsybl/powsybl-deployment.git
+cd powsybl-deployment
+```
+
+Install the orchestration tool docker-compose then: 
+```bash 
+cd docker-compose/
+docker-compose up
+```
+Note : When using docker-compose for deployment, your machine is accessible from the containers thought the ip adress
+`172.17..0.1` so to make the cassandra cluster, running on your machine, accessible from the deployed
+containers change the '<YOUR_IP>' of the first section to `172.17.0.1`
+
+You can now access to the application and the swagger UI of all the Spring services:
+
+```bash 
+DOCKER-COMPOSE_IP=`localhost`
```

Comment:
In this readme as IP is fixed in case of docker compose (localhost) that would be easier to read to just replace DOCKER-COMPOSE-IP by localhost